src/runner/lowlevel_keyboard_event.cpp

```diff
@@ -18,7 +18,15 @@ namespace {
   }
 }
 
+ //#define DISABLE_LOWLEVEL_HOOK // Prevent system-wide input lagging while paused in the debugger

```

Comment:
Two nits:
- move the comment above the define
- make the name more verbose, like `DISABLE_LOWLEVEL_KBHOOK_WHEN_DEBUGGED`